test/fuzzer/duckfuzz/array_slice_underflow.test

```diff
@@ -5,12 +5,8 @@
 statement ok
 PRAGMA enable_verification
 
-query I
+statement error
 SELECT ([1, 2, 3])[0:-9223372036854775808];
-----
-NULL
 
-query I
+statement error
 SELECT ([1, 2, 3])[-9223372036854775808:0];
-----
-NULL
```

Comment:
Just a nit. We might want to compare error messages here, to make sure that we don't get an `InternalException` or something.